code/obj/flock/structure/relay.dm

```diff
@@ -71,18 +67,16 @@
 	if(elapsed >= charge_time_length/2)
 		if(icon_state == "structure-relay")
 			icon_state = "structure-relay-glow"
-		// halfway point, start playing radio noises at people too
```

Comment:
useful comment

Reply:
I think halfway point is explained by the if() above, think the radio noises probably explained below too?